### DISCUSSION TITLE
Scope in query

### DISCUSSION THREAD

**Discussion by david-lobo** [statamic/cms#6168] [Q&A]
I have a query scope being used inside a collection tag in Antlers, but i want to make a custom tag instead with the query in the PHP code.  Is there a way to use a scope inside a PHP collection/entry query?

This is the Scope:

App\Scopes\EventEntries

And this is the query i am trying to use:

        $entries = Entry::query()
        ->where('collection', 'events')
        ->eventEntries()
        ->limit(5)
        ->get();

**Comment by jasonvarga** [comment]
#5927 is aiming to make it nicer to do that, but in the mean time you could do this:

```php
$query = Entry::query()
    ->where('collection', 'events');
    ->limit(5);

(new EventEntries)->apply($query, []);

$entries = $query->get();
```
